README.md

```diff
@@ -37,118 +37,9 @@ If you’re looking to contribute, have questions, or want to keep up-to-date ab
-
-## Λrrow Core
-
-```groovy
-def arrow_version = "0.10.1-SNAPSHOT"
-dependencies {
-    compile "io.arrow-kt:arrow-core:$arrow_version"
-    compile "io.arrow-kt:arrow-syntax:$arrow_version"
-    kapt    "io.arrow-kt:arrow-meta:$arrow_version"
-}
-```
-
-## Λrrow Core + Λrrow Optics
-
-```groovy
-def arrow_version = "0.10.1-SNAPSHOT"
-dependencies {
-    compile "io.arrow-kt:arrow-optics:$arrow_version"
-    compile "io.arrow-kt:arrow-syntax:$arrow_version"
-    kapt    "io.arrow-kt:arrow-meta:$arrow_version"
-}
-```
-
-## Λrrow Core + Λrrow Fx 
-
-```groovy
-def arrow_version = "0.10.1-SNAPSHOT"
-dependencies {
-    compile "io.arrow-kt:arrow-fx:$arrow_version"
-    compile "io.arrow-kt:arrow-syntax:$arrow_version"
-    kapt    "io.arrow-kt:arrow-meta:$arrow_version"
-}
-```
-
-## Λrrow Core + Λrrow Optics + Λrrow Fx
-
-```groovy
-def arrow_version = "0.10.1-SNAPSHOT"
-dependencies {
-    compile "io.arrow-kt:arrow-fx:$arrow_version"
-    compile "io.arrow-kt:arrow-optics:$arrow_version"
-    compile "io.arrow-kt:arrow-syntax:$arrow_version"
-    kapt    "io.arrow-kt:arrow-meta:$arrow_version"
-}
-```
-
-Here is the complete [library list](https://arrow-kt.io/docs/quickstart/libraries/) for a more granular dependency set-up.
-
-# Additional Setup
-
-For projects that wish to use their own `@higherkind`, `@optics` and other meta programming facilities provided by Λrrow
-the setup below is also required:
-
-Add the dependencies into the project's `build.gradle`
-
-```groovy
-apply plugin: 'kotlin-kapt' //optional
-apply from: rootProject.file('gradle/generated-kotlin-sources.gradle') //only for Android projects
-
-def arrow_version = "0.10.1-SNAPSHOT"
-dependencies {
-    ...
-    kapt    'io.arrow-kt:arrow-meta:$arrow_version' //optional
-    ...
-}
-```
-
-`gradle/generated-kotlin-sources.gradle`
-```groovy
-apply plugin: 'idea'
-
-idea {
-    module {
-        sourceDirs += files(
-                'build/generated/source/kapt/main',
-                'build/generated/source/kapt/debug',
-                'build/generated/source/kapt/release',
-                'build/generated/source/kaptKotlin/main',
-                'build/generated/source/kaptKotlin/debug',
-                'build/generated/source/kaptKotlin/release',
-                'build/tmp/kapt/main/kotlinGenerated')
-        generatedSourceDirs += files(
-                'build/generated/source/kapt/main',
-                'build/generated/source/kapt/debug',
-                'build/generated/source/kapt/release',
-                'build/generated/source/kaptKotlin/main',
-                'build/generated/source/kaptKotlin/debug',
-                'build/generated/source/kaptKotlin/release',
-                'build/tmp/kapt/main/kotlinGenerated')
-    }
-}
-```
+# Setup
+
+Take a look at [Setup in the Arrow website](https://arrow-kt.io/docs/quickstart/setup/) or [Setup file](https://github.com/arrow-kt/arrow/tree/master/modules/docs/arrow-docs/docs/docs/quickstart/setup).
```

Comment:
I would keep a tl;dr version of the setup here, just a couple of lines here.

Reply:
And relative paths, if possible.

Reply:
Thanks for the inputs, @pakoito !! Sure, it's possible for the second one.